test/utils.test.sh

```diff
@@ -238,19 +239,34 @@ clear_dockerfile
 credentials ${tmpdocker}
 assert "doesNotContain" "${args}" "NPMAccessToken=${NPMAccessToken}"
 
+# credentials() no github token in env test
```

Comment:
GithubToken differs from NPMAccessToken in that the github token is a required env var -- you can't run conex without it. I think you should be able to remove this test, and simplify the condition under which you set the --build-arg in utils.sh